python/lsst/cp/pipe/ptc.py

```diff
@@ -145,6 +145,11 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Sigma cut for outlier rejection in PTC.",
         default=5.0,
     )
+    sigmaCutAfwMathStatsControl = pexConfig.Field(
```

Comment:
This should probably be called somethingNumSigmaClip, so it's clear what the clipping is being applied to (the "something") and so it matches the name of the property set in the statsControl object.

Reply:
`nSigmaClipPtc` would, I think, be in keeping with our namings elsewhere.